src/helpers/bridge_contracts.js

```diff
@@ -437,6 +447,21 @@ export const ETH_BRIDGE_CONTRACTS = {
     ],
     ...PROJECT_BINANCE_SMART_CHAIN,
   },
+  [PROJECT_BOBA_NETWORK.name]: {
+    "bridges": [
+      {
+        "address": "0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c0",
```

Comment:
can't find such address in https://etherscan.io/search?f=0&q=0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c0 please confirm

Reply:
Thanks for picking up on that - I'd somehow left the last character off the end of the address. I've fixed that now.